### ISSUE TITLE
Distribute bitmap fonts in PIL format with the debian package

### ISSUE THREAD

**Issue by ddemidov** [rhempel/ev3dev-lang-python#223]
See @dlech's comments in ev3dev/ev3dev#751 on how to actually do this.

Besides packaging the fonts with the library, I would also provide a helper function for their easy consumption. Something like `font('<name>_<size>')` so that the following is possible:
```py
import ev3dev.ev3 as ev3
screen = ev3.Screen()
screen.draw.text('Hello', font=ev3.font('<name>_<size>'))
```

@WasabiFan, I guess this may wait for after we release 0.7.0?

**Comment by WasabiFan** [comment]
> I guess this may wait for after we release 0.7.0?

You have six hours :wink:

But more seriously, yes I think it should wait. That doesn't mean it can't be released soon, though; we could release 0.7.1 tomorrow if you'd like. I support the idea, I just want to make sure that we get this release finished at some point.

**Comment by ddemidov** [comment]
> You have six hours

My phone says it will wake me up for work in 7 hours and 15 minutes. Lets wait for 0.7.1 :sleeping:

**Comment by ddemidov** [comment]
Notes to myself:

* https://packages.debian.org/en/sid/all/xfonts-75dpi/download
* https://github.com/ganaware/pcf2bdf
* http://stackoverflow.com/questions/1395593/managing-resources-in-a-python-project

**Comment by ensonic** [comment]
FYI, thats what I do for openroberta right now:
https://github.com/OpenRoberta/robertalab-ev3dev/blob/develop/roberta/ev3.py#L46

**Comment by ddemidov** [comment]
Resolved by #225
